project/src/components/app/app.tsx

```diff
@@ -10,7 +21,31 @@ const FILM_INFO = {
 
 function App(): JSX.Element {
   return (
-    <Main name={FILM_INFO.name} genre={FILM_INFO.genre} release={FILM_INFO.release} posterImage={FILM_INFO.posterImage} backgroundImage={FILM_INFO.backgroundImage}/>
+    <BrowserRouter>
+      <Switch>
+        <Route path="/" exact>
+          <Main name={FILM_INFO.name} genre={FILM_INFO.genre} release={FILM_INFO.release} posterImage={FILM_INFO.posterImage} backgroundImage={FILM_INFO.backgroundImage}/>
+        </Route>
+        <Route path="/login">
+          <SignIn/>
+        </Route>
+        <PrivateRoute isAuthenticated={isAuthenticated} authenticationPath={'/login'}>
```

Comment:
Нужно добавить path="/list" exact
иначе всегда будет открываться список.

```suggestion
        <PrivateRoute isAuthenticated={isAuthenticated} authenticationPath={'/login'} path="/list" exact>
```